raiden/tests/integration/network/transport/rif_comms/test_client.py

```diff
@@ -347,3 +520,79 @@ def test_unsubscribe_from_peers(comms_clients):
     client_2.unsubscribe_from(client_1.rsk_address.address)
     assert client_2._is_subscribed_to(client_1.rsk_address.address) is False
     assert client_1._is_subscribed_to(client_2.rsk_address.address) is False  # check operations are independent
+
+
+def test_client_timeouts():
+    nodes = []
+    try:
+        node_1 = CommsNode(CommsConfig(node_id="A", amount_of_clients=1, auto_connect=False))
+        nodes.append(node_1)
+        client_1, address_1 = node_1.clients[0], node_1.clients[0].rsk_address.address
+
+        node_2 = CommsNode(CommsConfig(node_id="B", amount_of_clients=1, auto_connect=False))
+        nodes.append(node_2)
+        client_2, address_2 = node_2.clients[0], node_2.clients[0].rsk_address.address
```

Comment:
do we really need 2 clients here?

if the operations are supposed to be timing out, it doesn't really matter if—for example—the second client's address is in the DHT anyway, no?

(maybe i am missing something... :thinking:)

Reply:
Actually we could remove it yes because that it's an old logic, now it's just to get the receiver address for the send message operation, i will remove that yes.